README.md

```diff
@@ -26,6 +26,11 @@ Examples:
     console.log(randomWords({ exactly: 2 }));
     ['beside', 'between']
 
+    console.log(randomWords({ min: 2, max: 3, seed: 'my-seed' }));
+    console.log(randomWords({ min: 2, max: 3, seed: 'my-seed' }));
```

Comment:
I'm not sure why this is repeated twice. Can you please remove one?

Oh, wait...Now I understand why you included it twice. To clarify, add a comment describing why you added the example twice.

Reply:
Added comment
> // using a seed yields the same random words